### DISCUSSION TITLE
Use a checkbox to generate the date the checkbox was clicked.

### DISCUSSION THREAD

**Discussion by g1ronn1mo** [RafaelGB/obsidian-db-folder#989] [General]
Hey, is there a way to show the date a checkbox was clicked? 

If I finish a file, I want to mark it as done and automatically set a date in an other column, when it was done. 

Best regards

**Comment by Jacaran** [comment]
I would very much like this finction as well.
(I'm manually clicking checkboxs while also adding the dates.)
